addons/sale_timesheet/views/project_task_views.xml

```diff
@@ -8,11 +8,6 @@
         <field name="arch" type="xml">
             <div class="oe_button_box" position="inside">
                 <button string="Project Overview" class="oe_stat_button" type="object" name="action_view_timesheet" icon="fa-puzzle-piece" attrs="{'invisible': [('allow_billable', '=', False)]}"/>
-                <button class="d-none d-md-inline oe_stat_button"
-                        type="object" name="action_view_so" icon="fa-dollar"
-                        attrs="{'invisible': ['|', '|', ('allow_billable', '=', False), ('sale_order_id', '=', False), ('bill_type', '!=', 'customer_project')]}"
-                        string="Sales Order"
-                        groups="sales_team.group_sale_salesman"/>
```

Comment:
Have removed that record will definitely remove the correct attrs in sale_timesheet. You should have kept a record here that was inheriting from the one in sale_project and set the correct attrs.

Reply:
After discussing with the PO, it was decided that the button should be visible/accessible without the need for this module (and thus not using these attrs, which are not present in the sale_project module). Should it still use these attrs whenever sale_timesheet is installed ?

Reply:
Hello @proose,

We have told @aju-odoo that the move done implied the attrs loss in sale_timesheet as no inheritance has been put in place in sale_timesheet in this fix. She did not expected that and said that this should indeed be fixed.

Thanks and have a great day off!

Reply:
Hello @IT-Ideas,
Here is the link to the PR for the fix's fix: https://github.com/odoo/odoo/pull/74102 (I will also ping you on it)
Please tell me if that is what you wanted.

Have a great day!